Projects/DbCLI/CreateDb.py

```diff
@@ -69,84 +70,88 @@
 from rdkit.Chem.MolDb import FingerprintUtils
 
 # ---- ---- ---- ----  ---- ---- ---- ----  ---- ---- ---- ----  ---- ---- ---- ----
-from optparse import OptionParser
-parser = OptionParser(_usage, version='%prog ' + _version)
-parser.add_option('--outDir', '--dbDir', default='', help='name of the output directory')
-parser.add_option('--molDbName', default='Compounds.sqlt', help='name of the molecule database')
-parser.add_option('--molIdName', default='compound_id', help='name of the database key column')
-parser.add_option('--regName', default='molecules', help='name of the molecular registry table')
-parser.add_option('--pairDbName', default='AtomPairs.sqlt', help='name of the atom pairs database')
-parser.add_option('--pairTableName', default='atompairs', help='name of the atom pairs table')
-parser.add_option('--fpDbName', default='Fingerprints.sqlt',
+import argparse
+
+parser = argparse.ArgumentParser(usage)
+
+parser.add_argument('filename')
+parser.add_argument('-v', '--version', action='version', version='0.13.0')
```

Comment:
Interesting, I've never used that before. Instead of defining the version as a string, use the variable that is defined further above.

Reply:
`'version'` - *This expects a version = keyword argument in the add_argument() call, and prints version information and exits when invoked.*

Tried using the defined string, but that did not work - 
```
_version = "3.1.0"

import argparse
parser = argparse.ArgumentParser()
args = parser.parse_args()

parser.add_argument('-v', '--version', action='version' + _version)
```
Error - raise ValueError('unknown action "%s"' % (action_class,))

Reply:
You added it to the `action` keyword argument not as a separate `version` keyword argument
```
parser.add_argument('-v', '--version', action='version', version=version)
```

Reply:
Tried this:
```
version = "3.1.0"

import argparse
parser = argparse.ArgumentParser()
args = parser.parse_args()

parser.add_argument('-v', '--version', action='version', version =version)
```
Error:
Someshs-MacBook-Air:rdkit Somesh$ python3.6 check2.py -v
usage: check2.py [-h]
check2.py: error: unrecognized arguments: -v

Someshs-MacBook-Air:rdkit Somesh$ python3.6 check2.py --version
usage: check2.py [-h]
check2.py: error: unrecognized arguments: --version

Reply:
Check the order of the statements in your code